app/src/main/java/com/randomappsinc/studentpicker/utils/PreferencesManager.java

```diff
@@ -99,9 +99,11 @@ public boolean hasSeenPremiumTooltip() {
         return hasSeenTooltip;
```

Comment:
I forgot to update this to make it more readable :(

Reply:
No worries, we can worry about tooltip code later.